### PR TITLE
#161470738 Migrate project setup's flask script to flask cli

### PR DESCRIPTION
**Pull request by mutoromike**
## Description

  - Removes deprecated Flask-Script and replaces it with Flask-Cli
  - Removes hard-coded token info from the backend

## Fix

  - Creates a FLASK_APP in manage.py
  - Removes Manager, sets up Flaskgroup
  - Configures tests to now use an instance of the flask application
  - Sets up all docker images to point to the FLASK_APP
  - Removes all sample token that was being cross-checked for user roles

## How to test

- Clone the repository, install docker, run `make test` within the repository.
- Trigger a build on CircleCI.

### CODE REVIEW

File: src/requirements.txt [hunk 1]
```diff
@@ -1,40 +1,61 @@
...
-atomicwrites==1.2.1
-attrs==18.2.0
-autopep8==1.4
+appnope==0.1.0
+astroid==2.0.4
+Babel==2.6.0
+backcall==0.1.0
+billiard==3.5.0.3
 blinker==1.4
+celery==4.2.1
```
Comment (georgreen) [inline src/requirements.txt]: Remove this.

File: src/requirements.txt [hunk 2]
```diff
@@ -43,8 +64,16 @@ python-jose==2.0.2
...
-six==1.10.0
+simplegeneric==0.8.1
+six==1.11.0
 SQLAlchemy==1.1.14
+tornado==5.1
+traitlets==4.3.2
+typed-ast==1.1.0
 urllib3==1.22
+vine==1.1.4
+virtualenv==16.0.0
```
Comment (georgreen) [inline src/requirements.txt]: Remove this.

Comment (mutoromike) [reply src/requirements.txt]: 👍

File: src/requirements.txt [hunk 3]
```diff
@@ -1,40 +1,61 @@
...
 chardet==3.0.4
 click==6.7
 coverage==4.5
 coveralls==1.2.0
+decorator==4.3.0
 docopt==0.6.2
 ecdsa==0.13
-Flask==0.12.2
+flake8==3.6.0
+Flask==0.12.4
```
Comment (WinstonKamau) [inline src/requirements.txt:19]: Hey, @mutoromike  I wanted to suggest that for dependencies that were there before, we stick to their former version unless we want to intentionally change it after confirming that the version we are changing to is stable for our application.

Comment (mutoromike) [reply src/requirements.txt:19]: Thanks for the concern, this was changed due to Flask-CLI's requirements.

Comment (WinstonKamau) [reply src/requirements.txt:19]: In that case, then we can keep it that way 👍

**Review by WinstonKamau — APPROVED** [review]

File: src/requirements.txt (hunk 3 above)
Comment (mutoromike) [reply src/requirements.txt:19]: 👍

**Comment by WinstonKamau** [comment]
Hey, @mutoromike you should add the new environment variable to the README of the sandbox and the README on the root of the repository.

**Review by jimmykamau — APPROVED** [review]

**Pull request merged**